great_expectations/data_context/data_context.py

```diff
@@ -3251,6 +3251,16 @@ def run_checkpoint(
             **kwargs,
         )
 
+    def list_profilers(self) -> List[str]:
+        if self.profiler_store is None:
+            raise ge_exceptions.StoreConfigurationError(
+                f"Attempted to list profilers from a Profiler Store, which is not a configured store."
+            )
+        return profiler_toolkit.list_profilers(
```

Comment:
What is the `profiler_toolkit` and is it some sort of global?

Reply:
It's a utility file of sorts to encapsulate the behavior that is called in the DataContext. 

```python
from great_expectations.rule_based_profiler import toolkit # toolkit.py
```
I mirrored the implementation of Checkpoints: https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/data_context/data_context.py#L3099-L3103

I mainly used it to adhere to existing patterns but I also like separating the logic out to simplify the ever growing DataContext.

Reply:
@cdkini  the `toolkit` has already been helpful in the CLI-upgrade task that I've been working on :) Thank you for it